apps/els_lsp/test/els_hover_SUITE.erl

```diff
@@ -526,13 +540,15 @@ nonexisting_type(Config) ->
     #{result := Result} = els_client:hover(Uri, 22, 15),
     %% The spec for `j' is shown instead of the type docs.
     Value =
-        case has_eep48_edoc() of
-            true ->
+        case list_to_integer(erlang:system_info(otp_release)) of
+            25 ->
                 <<
-                    "## j/1\n\n---\n\n```erlang\n\n  j(_) \n\n```\n\n"
-                    "```erlang\n-spec j(doesnt:exist()) -> ok.\n```"
```

Comment:
I think there is a bug in `els_eep48_docs:render` when running in OTP versions under 25, and that is why this current test code looks a bit weird (it checks if has_eep48_edoc() is true or false but uses the same test data regardless)

On OTP 24 locally on my machine, `els_eep48_docs:render` returns `{error, function_missing}` for this `j` function that is hovered. On OTP 25 it returns some kind of doc chunks instead even of the function is not documented. Maybe OTP started automatically adding doc chunks for specs?

Anywho, I didn't want to make changes to the production code in this PR adding OTP 25 to CI, so not diving into if `els_eep48_docs:render` should be improved (especially since it seems to me that it's under OTP 24 it is behaving badly).